src/context.cpp

```diff
@@ -407,22 +407,45 @@ namespace librealsense
     //A facade for a realdds::dds_option exposing librealsense interface
     class rs2_dds_option : public option_base
     {
-        std::shared_ptr< realdds::dds_option > _dds_opt;
-        std::string _stream_name;
-
     public:
-        rs2_dds_option( const std::shared_ptr< realdds::dds_option > & dds_opt )
+        typedef std::function< void( const std::string & name, float value ) > set_option_callback;
```

Comment:
The value is const too right?

Reply:
`value` is passed by value, that is, a copy of the original value, so it does not change

Reply:
Yes I see, from what I know, as best practice, if a value should not be changed inside a function (even if it's a local copy) you add a `const` specifier so someone will not accidently write on it.
But as I read online it's a matter of style.

That's OK